### DISCUSSION TITLE
S.O.S Suspected iPhone hacked. Please help

### DISCUSSION THREAD

**Discussion by RoyGood1888** [community/community#33492] [Code Security]
I don’t know if this is an inappropriate platform to ask for this kind of help but I am desperate. I have reason to believe that my iPhone is being attacked. This is the second phone in less than two years. I have been trying to learn how to read and understand the stored data analytics but need help to confirm or deny what is or is not malicious code so I don’t just jump to conclusions. If anyone is willing to help me I would greatly appreciate it and I would even be willing to pay for the time spent. Thank you

**Comment by ghost** [comment]
Did you ever get an answer, I'm positive mines been hacked, my ex had my phone in his hand when I got up one morning, since then I have accounts I never created, he started using my phone number for all his online hotel accounts, whatsapp, line, tinder, he was using it to to call the woman he was cheating on me with, I have the outgoing call logs to HIS contacts in my phone logs. He even created this account which is why I'm wondering what he uses it for. I have changed my number made 10 or more new email accounts but he just keeps getting access. I've factory reset everything, I cant get him out and the law does not give a sh*t. I need help please, I'm on here tonight because I came accross a beef/config.yaml in my personal search results in google. It looks like its a browser setup to retrieve passwords. I dont understand any of this, please HeLP
